Makefile

```diff
@@ -119,6 +119,7 @@ test-go: libflux-go
 
 test-rust:
 	cd libflux && $(CARGO) test $(CARGO_ARGS) --all-features && \
+	$(CARGO) check && \
```

Comment:
`cargo test` which is called on the line above here should cover everything that `cargo check` does already

Reply:
This does work. Thanks @Marwes!